Branch-SDK/src/io/branch/referral/Defines.java

```diff
@@ -60,6 +60,10 @@
         Language("language"),
         IsReferrable("is_referrable"),
         Update("update"),
+        OriginalInstallTime("original_install_time"),
+        FirstInstallTime("first_install_time"),
```

Comment:
original vs first is pretty unclear. Perhaps the following for clarity:
`original_install_time` -> `first_install_time`
`first_install_time` -> `latest_install_time`

Reply:
unfortunately Android classes calla latest_install_time as `first_install_time`. We added `original_install_time` to avoid the confusion. Do you think that make sense

Reply:
I see what it's called in the class, but this is our own internal reference. That means the data team will have to understand what it means, not necessarily other Android people. Because `original` and `first` are basically the same thing, I think it'd be helpful to change the description to reflect the functionality.

Reply:
Make sense, will change this